dali/pipeline/operators/geometric/bb_flip_test.cc

```diff
@@ -169,13 +178,18 @@ std::vector<Arguments> arguments = {
         {{"horizontal", 0}, {"vertical", 0}},
 };
 
+std::vector<Arguments> devices = {
+    {{"device", std::string{"cpu"}}},
+    {{"device", std::string{"gpu"}}},
+};
```

Comment:
Shouldn't
```
std::vector<Arguments> devices = {
    {{"device", "cpu"}},
    {{"device", "gpu"}},
};
```
suffice?

Reply:
I believe it doesn't. I remember having to add it myself. Maybe something related to that std::any implementation?

Reply:
No, it gets deducted as char[4] by TestOpArgValueImpl.

Reply:
...then _maybe_ we should modify the Argument to convert const char* to std::string. I think it's unlikely that someone will pass an array of `char`s and expect it to be stored as an array - especially given the fact that `char` is special - neither signed nor unsigned.